### PR TITLE
feat(netguard): remove-broadcasts-restriction

### PR DESCRIPTION
**Pull request by Linus789**
Makes it possible to start/stop [NetGuard](https://github.com/M66B/NetGuard) via automation apps. Example for ADB to start NetGuard:
```
adb shell am broadcast --user 0 -a eu.faircode.netguard.ON eu.faircode.netguard
```

### CODE REVIEW

**Comment by oSumAtrIX** [comment]
NetGuard is open source; why would you want to patch it if you can precompile from source?

**Comment by Linus789** [comment]
Because it is a lot easier just to patch it. It is only a single Manifest that is modified, so there is no compelling reason for me to recompile the entire app.

**Comment by Linus789** [comment]
I don’t think that a PR like this would be accepted, because the ability for third-party-apps to stop NetGuard might impose a security risk.

File: src/main/kotlin/app/revanced/patches/netguard/broadcasts/removerestriction/resource/patch/RemoveBroadcastsRestrictionPatch.kt [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+@Patch(false)
+@Name("remove-broadcasts-restriction")
+@Description("Enables starting/stopping NetGuard via broadcasts.")
+@RemoveBroadcastsRestrictionCompatibility
+@Version("0.0.1")
+class RemoveBroadcastsRestrictionPatch : ResourcePatch {
+    override fun execute(context: ResourceContext): PatchResult {
+        // create an xml editor instance
+        context.xmlEditor["AndroidManifest.xml"].use { dom ->
+            // get the application node
```
Comment (oSumAtrIX) [inline src/main/kotlin/app/revanced/patches/netguard/broadcasts/removerestriction/resource/patch/RemoveBroadcastsRestrictionPatch.kt:23]:
```suggestion
        context.xmlEditor["AndroidManifest.xml"].use { dom ->
```

**Review by oSumAtrIX — APPROVED** [review]

**Pull request merged**